tools/export.py

```diff
@@ -190,21 +190,25 @@ def symbolic(self, g, featmap):
                     flatten_i=1,
                     outputs=self.num_outputs)
 
-
-def main(args):
-    model = init_detector(args.config, args.checkpoint, device="cpu")
-    model.eval().cuda()
-    torch.set_default_tensor_type(torch.FloatTensor)
-
-    if isinstance(model.bbox_head, AnchorHead):
-        anchor_generators = model.bbox_head.anchor_generators
+def stub_anchor_generator(anchor_head):
+    if anchor_head is not None and isinstance(anchor_head, AnchorHead):
+        anchor_generators = anchor_head.anchor_generators
         for i in range(len(anchor_generators)):
             anchor_generators[i].grid_anchors = AnchorsGridGeneratorStub(
                 anchor_generators[i].grid_anchors)
             # Save base anchors as operation parameter. It's used at ONNX export time during symbolic call.
             anchor_generators[i].grid_anchors.params['base_anchors'] = anchor_generators[
                 i].base_anchors.cpu().numpy()
 
+def main(args):
+    model = init_detector(args.config, args.checkpoint, device="cpu")
+    model.eval().cuda()
+    torch.set_default_tensor_type(torch.FloatTensor)
+
+    if not args.no_anchor_stub:
+        stub_anchor_generator(model.rpn_head)
```

Comment:
looks like it does not work for models that do not have rpn

Reply:
Yeah, fixed.